public/pages/Dashboard/components/FindingsDashboard/FindingFlyout.js

```diff
@@ -70,10 +70,16 @@ export default class FindingFlyout extends Component {
 
   closeFlyout = () => {
     this.setState({ isFlyoutOpen: false });
+
+    const { dashboardFlyoutIsOpen = false, closeFlyout } = this.props;
+
+    if (typeof closeFlyout === 'function') {
+      if (dashboardFlyoutIsOpen) closeFlyout();
```

Comment:
club the two if conditions?

Reply:
I've used defensive mimicking on already existing idea few lines above. I could actually simplify.
<img width="703" alt="image" src="https://user-images.githubusercontent.com/1564068/204912564-efbe7cf4-62be-41ad-a416-574b409235cf.png">

Reply:
fixed